lib/vgmstream/kodi-note.txt

```diff
@@ -0,0 +1,2 @@
+vgmstream source from https://github.com/losnoco/vgmstream
+Sync to r1040
```

Comment:
From my side it would be good to include this note file always with integration of complete external sources and should be obligatory for all with us.

Before I found it very hard to find out the versions and sources used on such addons.